File: Assignment_11/program1.c
// Input 5

/*
    A       B       C       D       E
*/

#include<stdio.h>

int Pattern(int iNo)
{
    int iCnt = 0;
    char ch = '\0';

    if(iNo < 0)
    {
        iNo = -iNo;
    }

    for(iCnt = 1, ch = 'A'; iCnt <= iNo; iCnt++, ch++)
    {
        printf("%c\t",ch);
    }
}

int main()
{
    int iValue = 0;

    printf("Enter Number of Elements : ");
    scanf("%d",&iValue);

    Pattern(iValue);

    return 0;
}

//  Time Complexity = O(N)

File: Assignment_13/program1.java
// Input 4 4

/*
    A       B       C       D
    A       B       C       D
    A       B       C       D
    A       B       C       D
*/
import java.util.*;

class Pattern
{
    public void Display(int iRow, int iCol)
    {
        int i = 0, j = 0;
        char ch = '\0';

        for(i = 1; i <= iRow; i++)
        {
            for(j = 1, ch = 'A'; j <= iCol; j++, ch++)
            {
                System.out.print(ch+"\t");
            }
            System.out.println();
        }
    }
}

class program1
{
    public static void main(String A[])
    {
        int iValue1 = 0, iValue2 = 0;

        Scanner sobj = new Scanner(System.in);

        System.out.print("Enter rows and columns: ");
        iValue1 = sobj.nextInt();
        iValue2 = sobj.nextInt();

        Pattern pobj = new Pattern();

        pobj.Display(iValue1,iValue2);
    }
}

//  Time Complexity = O(N)